src/parser/jobs/sge/modules/Overheal.tsx

```diff
@@ -9,23 +10,30 @@ export class Overheal extends CoreOverheal {
 	override displaySuggestion = true
 
 	override suggestionIcon = this.data.actions.DIAGNOSIS.icon
+	override overhealName = <Trans id="sge.overheal.direct.name">GCD Heals</Trans>
 
 	override trackedHealCategories = [
 		{
 			name: <Trans id="sge.overheal.addersgall.name">Addersgall</Trans>,
```

Comment:
Not worth blocking over, but since this doesn't appear anywhere anymore, it might be worth replacing this with plain `<></>` tags instead of Trans tags, but just my 2¢.

Reply:
If it doesn't appear anywhere, why does it still exist?

Reply:
As we discussed on Discord, I'd left it as-is due to how the core Overheal module's debug output worked, but we figured out a plan to revise that so I could change the bucket name to just be a string.